openstef_dbc/utils.py

```diff
@@ -57,3 +57,34 @@ def parse_influx_result(
     result.index.name = "datetime"
     result.columns.name = ""
     return result
+
+
+def round_time_difference(time_diff, options):
+    """Rounds a time difference to the first bigger option.
+
+    Used to calculate the tAhead for forecasts"""
+    next_bigger_option = next((opt for opt in options if opt > time_diff), None)
+    if next_bigger_option is None:
+        return time_diff
+    return next_bigger_option
```

Comment:
This function is exactly the same as `round_single_time_diff` and is not used, or am I missing something?

Reply:
You are right. I copied parts of stuff twice